README.md

```diff
@@ -21,12 +21,18 @@
 
 ```bash
 yarn install
-NODE_ENV=development node_modules/.bin/webpack-cli --mode development
+yarn build
 ```
 #### Enable debug mode
 - `touch local_settings.py`
 - Inside `local_setting.py`, have `DEBUG = True`
 
+If you experience missing locale, try add locale to your shell script
+
+#### Init Db (if doesnt exist)
```

Comment:
Should be:
#### Initialize the database (if it doesn't exist)

Reply:
Fixed in new commit.